apps/nextjs-example/postcss.config.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+const projectRoot = __dirname;
+const monorepoRoot = path.join(projectRoot, '../../');
+
+function getPackageIncludePaths(packageName, nodeModulePaths) {
+  let packagePath = null;
+
+  for (const nodeModulePath of nodeModulePaths) {
+    const packageJsonPath = path.resolve(
+      nodeModulePath,
+      packageName,
+      'package.json',
+    );
+    if (fs.existsSync(packageJsonPath)) {
+      packagePath = path.dirname(packageJsonPath);
+      break;
+    }
+  }
+  if (!packagePath) {
+    throw new Error(`Could not find package ${packageName}`);
+  }
+
+  return [
+    path.join(packagePath, '**/*.{js,mjs}'),
+    '!' + path.join(packagePath, 'node_modules/**/*.{js,mjs}'),
+  ];
+}
+
+const openPropsIncludePaths = getPackageIncludePaths('@stylexjs/open-props', [
+  path.join(projectRoot, 'node_modules'),
+  path.join(monorepoRoot, 'node_modules'),
+]);
+
+module.exports = {
+  plugins: {
+    '@stylexjs/postcss-plugin': {
+      include: [
+        'app/**/*.{js,jsx,ts,tsx}',
+        'components/**/*.{js,jsx,ts,tsx}',
+        ...openPropsIncludePaths,
+      ],
```

Comment:
Just so I can verify how the postcss plugin works:

We're fining all files that match these globs and compiling them as a list.

Since we're reading all the files manually anyway, if we wanted to we could accept a list of "entrypoints" instead and resolve the dependency graph, right?

Reply:
> Since we're reading all the files manually anyway, if we wanted to we could accept a list of "entrypoints" instead and resolve the dependency graph, right?

Yes. The postcss plugin globs and compile all matching files to extract css.
This means we may be over-compiling in some cases similar to how Tailwind CLI does it with the `content` glob.

We can accept entry points and resolve dependency graphs  ourselves but we need to take into account aliases, platform extensions (.web.ts), etc. to replicate bundler dependency graphs so I avoided that to allow greater flexibility to users and to be bundler-agnostic in general.